bin/hbase

```diff
@@ -766,13 +767,6 @@ elif [ "$COMMAND" = "copyreppeers" ] ; then
   CLASS='org.apache.hadoop.hbase.replication.CopyReplicationPeers'
 else
   CLASS=$COMMAND
-if [[ "$CLASS" =~ .*IntegrationTest.* ]] ; then
```

Comment:
should we be really dropping this, need to think more on this!

Reply:
Good point.

I would the option in the script, and make sure that ITs can be run from compiled source directory.

Reply:
Okay so basic we want to retain the 3 files under HBASE_HOME/lib/test i.e.

- hamcrest-core-1.3.jar
- junit-4.13.2.jar
- mockito-core-4.11.0.jar

Let me make this change and figure out if we can still run IT by building from source.

Reply:
Fixed